virttest/ssh_key.py

```diff
@@ -43,8 +55,8 @@ def get_public_key():
 
     else:
         logging.info('Neither RSA nor DSA keypair found, creating RSA ssh key pair')
-        process.system('ssh-keygen -t rsa -q -N "" -f %s' %
-                       rsa_private_key_path)
+        process.system("su - %s -c 'ssh-keygen -t rsa -q -N \"\" -f %s'" %
```

Comment:
Do we need judge whether client_user is None or not ? it sound likes the string will be alike 'su - None' if client_user is None

Reply:
Yes, you are right, updated.